pages/apis/rest_api/pipeline_templates.md

```diff
@@ -0,0 +1,293 @@
+      "name": "Sam Kim",
+      "email": "sam@example.com",
+      "avatar_url": "https://www.gravatar.com/avatar/example",
+      "created_at": "2013-05-03T04:17:55.867Z"
+    },
+    "updated_at": "2023-06-12T04:17:55.867Z",
+    "updated_by": {
+      "id": "3d3c3bf0-7d58-4afe-8fe7-b3017d5504de",
+      "graphql_id": "VXNlci0tLTNkM2MzYmYwLTdkNTgtNGFmZS04ZmU3LWIzMDE3ZDU1MDRkZQo=",
+      "name": "Sam Kim",
+      "email": "sam@example.com",
+      "avatar_url": "https://www.gravatar.com/avatar/example",
+      "created_at": "2013-05-03T04:17:55.867Z"
+    }
+  }
+]
+```
+
+Required scope: `read_pipeline_templates`
+
+Success response: `200 OK`
+
+## Get a pipeline template
+
+```bash
+curl "https://api.buildkite.com/v2/organizations/{org.slug}/pipeline-templates/{uuid}"
+```
+
+```json
+{
+  "uuid": "018a86cc-db73-7d15-8c68-5023cf8d64c3",
+  "graphql_id": "UGlwZWxpbmVUZW1wbGF0ZS0tLTAxOGE4NmNjLWRiNzMtN2QxNS04YzY4LTUwMjNjZjhkNjRjMw==",
+  "name": "Build template",
+  "description": "Shared build steps configuration",
+  "configuration": "steps:\n  - label: \":hammer: Build\"\n    command: \"scripts/build.sh\"",
+  "available": false,
+  "url": "http:///api.buildkite.com/v2/organizations/acme-inc/pipeline-templates/018a86cc-db73-7d15-8c68-5023cf8d64c3",
+  "web_url": "http://www.buildkite.com/organizations/acme-inc/pipeline-templates/018a86cc-db73-7d15-8c68-5023cf8d64c3",
+  "created_at": "2023-05-03T04:17:55.867Z",
+  "created_by": {
+    "id": "3d3c3bf0-7d58-4afe-8fe7-b3017d5504de",
+    "graphql_id": "VXNlci0tLTNkM2MzYmYwLTdkNTgtNGFmZS04ZmU3LWIzMDE3ZDU1MDRkZQo=",
+    "name": "Sam Kim",
+    "email": "sam@example.com",
+    "avatar_url": "https://www.gravatar.com/avatar/example",
+    "created_at": "2023-08-29T10:10:03.000Z"
+  },
+   "updated_at": "2023-06-12T04:17:55.867Z",
+  "updated_by": {
+    "id": "3d3c3bf0-7d58-4afe-8fe7-b3017d5504de",
+    "graphql_id": "VXNlci0tLTNkM2MzYmYwLTdkNTgtNGFmZS04ZmU3LWIzMDE3ZDU1MDRkZQo=",
+    "name": "Sam Kim",
+    "email": "sam@example.com",
+    "avatar_url": "https://www.gravatar.com/avatar/example",
+    "created_at": "2013-05-03T04:17:55.867Z"
+  }
+}
+```
+
+Required scope: `read_pipeline_templates`
+
+Success response: `200 OK`
+
+## Create a pipeline template
+
+```bash
+curl -X POST "https://api.buildkite.com/v2/organizations/{org.slug}/pipeline-templates" \
+  -H "Content-Type: application/json" \
+  -d '{
+    "name": ":hammer: Build",
+    "description": "Shared build steps configuration",
+    "configuration": "steps:\n  - label: \":hammer: Build\"\n    command: \"scripts/build.sh\"",
+    "available": true,
+  }'
+```
+
+```json
+{
+  "uuid": "018a86cc-db73-7d15-8c68-5023cf8d64c3",
+  "graphql_id": "UGlwZWxpbmVUZW1wbGF0ZS0tLTAxOGE4NmNjLWRiNzMtN2QxNS04YzY4LTUwMjNjZjhkNjRjMw==",
+  "name": "Build template",
+  "description": "Shared build steps configuration",
+  "configuration": "steps:\n  - label: \":hammer: Build\"\n    command: \"scripts/build.sh\"",
+  "available": false,
```

Comment:
This was set to `true` in the payload to create the template, should it also be true here?

```suggestion
  "available": true,
```